tests/units/DBmysqlIterator.php

```diff
@@ -762,12 +762,12 @@ public function testUnionQuery() {
       $union = new \QueryUnion($union_crit);
       $raw_query = 'SELECT * FROM ((SELECT * FROM `table1`) UNION ALL (SELECT * FROM `table2`))';
       $it = $this->it->execute(['FROM' => $union]);
-      $this->string($it->getSql())->isIdenticalTo($raw_query);
+      $this->string($it->getSql())->matches('/' . preg_quote($raw_query) . ' AS `union_[\w]{32}`/');
```

Comment:
Why not just add the preix to the expected raw query instead of a complicated regex?